src/Analysis/Engine/Impl/Values/ProtocolInfo.cs

```diff
@@ -268,85 +268,78 @@ public override IAnalysisSet Call(Node node, AnalysisUnit unit, IAnalysisSet[] a
 
         public override int GetHashCode() => ObjectComparer.Instance.GetHashCode(_protocols);
 
-        internal override bool UnionEquals(AnalysisValue av, int strength) {
-            if (av is ProtocolInfo pi) {
-                if (strength > 0) {
-                    return Name == pi.Name;
-                }
-                return ObjectComparer.Instance.Equals(_protocols, pi._protocols);
-            }
-            return false;
-        }
+        internal override bool UnionEquals(AnalysisValue av, int strength)
+            => av is ProtocolInfo pi ? ObjectComparer.Instance.Equals(_protocols, pi._protocols) : false;
 
-        internal override int UnionHashCode(int strength) {
-            if (strength > 0) {
-                return Name.GetHashCode();
-            }
-            return GetHashCode();
+    internal override int UnionHashCode(int strength) {
```

Comment:
This file's indention got all screwed up.

Reply:
Fixed